web-console/knikubevirt/openshift-vms-overview/openshift-vms-overview.md

```diff
@@ -0,0 +1,25 @@
+# OpenShift VMs Overview
+
+The purpose of this PR is to explore what Virtual Machines could look like in the Overview and Resources sections of Openshift/CNV.
+This is the first pass so really just trying to understand exactly what needs to be surfaced there.
+
+Through the Overview page the user can see the VM’s based on the application it is associated with via it’s label and and also see some sparkline charts that illustrate the usage of Network, CPU and Memory for the past 7 days.
+
+![sparkline](Openshift-overview-virtual-machine-sparkline.png)
+
+Clicking on the Kebab would offer a list of options depending on that state of the VM.
+
+![dropdown menu options](Openshift-overview-virtual-machine-dropdown.png)
+
+When the user clicks on the virtual machine it displays the side bar with 2 optional tabs: Overview and Resources. The selected item should have a blue highlight.
+
+## Overview tab
+
+![overview tab](Openshift-overview-virtual-machine-overviewtab.png)
```

Comment:
Something else I noticed in my latest review of the current implementation is that when clicking the row to show the side panel, it actually resizes the list view rather than overlaying on top of the list view. Here's a screenshot:
<img width="1415" alt="resize table" src="https://user-images.githubusercontent.com/208455/60918782-1f788e80-a262-11e9-94dc-88408bc9832f.png">